lang/fr.json

```diff
@@ -237,10 +237,10 @@
 "FAILED": "Échec!\n",
 "COMPLETED": "Complété!\n",
 "SETTINGS": "Options",
-"FORCE_UI_BOOT": "Forcer le démarrage de l'interphace graphique (GUI)    %ls",
+"FORCE_UI_BOOT": "Forcer le démarrage de l'interface graphique (GUI)    %ls",
```

Comment:
CAPS keys is from old translation - kept for compatibility until old console elimunated.